.golangci.yml

```diff
@@ -19,9 +18,11 @@ linters:
     - paralleltest
     - sqlclosecheck
     - staticcheck
-    - structcheck
     - unconvert
     - unused
+  disable:
+    - deadcode
+    - structcheck
     - varcheck
```

Comment:
this is left over from the attempts to get past linting? Should probably revert.

Reply:
I'm not 100% certain, but these errors made it sound like these have been effectively replaced by the `unused` linter:

```
Running [D:\a\_temp\2804da08-2f87-451c-b02a-aca27e802bd3\golangci-lint-1.52.2-windows-amd64\golangci-lint run --out-format=github-actions] in [] ...
[20](https://github.com/kolide/krypto/actions/runs/4796954099/jobs/8533333180#step:5:22)
  level=warning msg="[runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
[21](https://github.com/kolide/krypto/actions/runs/4796954099/jobs/8533333180#step:5:23)
  level=warning msg="[runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
[22](https://github.com/kolide/krypto/actions/runs/4796954099/jobs/8533333180#step:5:24)
  level=warning msg="[runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused."
```

Reply:
ahh, that makes sense